fidesctl/src/fidesctl/cli/commands/util_comands.py

```diff
@@ -0,0 +1,133 @@
+"""Contains all of the Utility-type CLI commands for Fidesctl."""
+import os
+
+import click
+import requests
+import toml
+
+from fidesctl.cli.options import yes_flag
+from fidesctl.cli.utils import (
+    handle_cli_response,
+    pretty_echo,
+)
+from fidesctl.core import api as _api
+from fidesctl.core.utils import echo_green, echo_red
+
+
+@click.command()
+@click.pass_context
+def init(ctx: click.Context) -> None:
+    """
+    Initialize a Fidesctl instance.
+    """
+
+    # Constants
+    dir_name = ".fides"
+    config_path = f"{dir_name}/fidesctl.toml"
+    config = ctx.obj["CONFIG"]
+
+    # List the values we want to exclude from the user-facing config
+    # These are values that are generated at config load time
+    excluded_values = {
+        "user": {"request_headers"},
+        "api": {"async_database_url", "sync_database_url"},
+    }
+
+    echo_green("Initializing Fidesctl...\n")
+
+    # create the .fides dir if it doesn't exist
+    if not os.path.exists(dir_name):
+        os.mkdir(dir_name)
+        echo_green(f"Created a '{dir_name}' directory.\n")
+    else:
+        echo_green(f"Directory '{dir_name}' already exists. Skipping...\n")
+
+    # create a config file if it doesn't exist
+    if not os.path.isfile(config_path):
+        config_url = "https://ethyca.github.io/fides/installation/configuration/"
```

Comment:
This is another constant that might come in handy if it were more widely available.
```suggestion
        config_docs_url = "https://ethyca.github.io/fides/installation/configuration/"
```

Reply:
updated the name, not going to move the scope for now though